### DISCUSSION TITLE
Change paramInformation seems to have no effect

### DISCUSSION THREAD

**Discussion by matteodisabatino** [zwave-js/zwave-js#5661] [General]
Hi everyone,

I created a simple code in order to force a device to log every minute. The code is:

```javascript
const node = this.driver.controller.nodes.get(31)
node.deviceConfig.paramInformation.set({ parameter: 111 }, { value: 60 })
```

I don't know if code is correct because I haven't found examples about that in the documentation. Anyway, it seems not work: the device still logs every hour.

Moreover, before executing the code below, the parameter configuration looks like that:
<img width="404" alt="Screenshot 2023-04-14 alle 10 33 38" src="https://user-images.githubusercontent.com/4496968/231990454-0ea6e255-634b-480b-828e-de09d563ed2e.png">

After the execution, instead, the result is that:
<img width="290" alt="Screenshot 2023-04-14 alle 10 33 47" src="https://user-images.githubusercontent.com/4496968/231990584-182d71df-be65-4f90-9d0d-bc2fa4776871.png">

What am I wronging?

**Comment by AlCalzone** [comment]
The `deviceConfig` property is meant as readonly information about the device.

What you're looking for is either the [`setValue`](https://zwave-js.github.io/node-zwave-js/#/api/node?id=setvalue) method, which needs the corresponding value ID, but automatically chooses the correct value size and format for the Z-Wave command. Or the [`set` method of the Configuration CC API](https://zwave-js.github.io/node-zwave-js/#/api/CCs/Configuration?id=set), which accepts a single argument of the following shape:
```ts
export type ConfigurationCCAPISetOptions = {
	parameter: number;
} & (
	| {
			// Variant 1: Normal parameter, defined in a config file
			bitMask?: undefined;
			value: ConfigValue;
	  }
	| {
			// Variant 2: Normal parameter, not defined in a config file
			bitMask?: undefined;
			value: ConfigValue;
			valueSize: 1 | 2 | 4;
			valueFormat: ConfigValueFormat;
	  }
	| {
			// Variant 3: Partial parameter, must be defined in a config file
			bitMask: number;
			value: number;
	  }
);
```
That one gets invoked like this (variant 1):
```ts
node.commandClasses.Configuration.set({
  parameter: 111,
  value: 60
}),
```

**Comment by matteodisabatino** [comment]
This is exactly what I needed. Thank you so much.
